src/Components/Shared/Table/index.module.css

```
@@ -0,0 +1,31 @@
+.buttonAdd{
```

```suggestion
.buttonAdd {
```
Review the spaces here. We should left an space before the `{` and we have to add a break line between each class.